.gitattributes

```diff
@@ -0,0 +1,34 @@
+# Make packagist / composer download smaller
+/.ddev export-ignore
+/.github export-ignore
+/dev
```

Comment:
```suggestion
/dev
/docs
```

what do you think about adding the /docs folder?

Reply:
Yeah, no one needs docs inside vendor